packages/node_modules/@webex/plugin-meetings/src/peer-connection-manager/index.js

```diff
@@ -163,7 +163,7 @@ pc.iceCandidate = (peerConnection, {remoteQualityLevel}) =>
 
       if (!validateSdp(peerConnection.sdp)) {
         setTimeout(() => {
```

Comment:
Could you add a comment around why this nested timeout is necessary. It is a bit difficult to understand.